pandas/core/frame.py

```diff
@@ -36,6 +37,7 @@
 
 import numpy as np
 import numpy.ma as ma
+import numpy.ma.mrecords as mrecords
```

Comment:
i think we import this lazily to speed up import time

Reply:
I thought maybe that was the case. However, looking at the PR that added the import (https://github.com/pandas-dev/pandas/pull/5579/files) I didn't see any discussion about import time. Do you know if there are any benchmarks to back this up I could run?

If it's a deal breaker, I can move the mrecords check into the ma check like it was in the original code.